src/lib/theme/default.ts

```diff
@@ -11,4 +11,29 @@ export default {
       base: 'py-5 px-5 last:rounded-b-lg dark:bg-gray-900',
     },
   },
+  alert: {
+    base: 'flex flex-col gap-2 p-4 text-sm',
+    borderAccent: 'border-t-4',
+    closeButton: {
+      base: '-mx-1.5 -my-1.5 ml-auto inline-flex h-8 w-8 rounded-lg p-1.5 focus:ring-2',
+      color: {
```

Comment:
I'm wondering if we don't want to replace the color names with some descriptive names... Do you know? Something like: `warning`, `info`, `default`, `success`... 'cause if the person decides to change the gree to use some red-ish color, kind of make it a little bit confusing.

Reply:
Yes! I think that would make the API a bit more intuitive as well